language_bindings/rust/rust_build/test_files/iir_blur.cpp

```diff
@@ -0,0 +1,178 @@
+// This file defines a generator for a first order IIR low pass filter
+// for a 2D image.
```

Comment:
It is a modified version accepting interleaved data. We could modify the example code to reformat to planner then the iir_blur generators would be the same. It may be better for demonstration to use another generator for this purpose IE a "convert_to_planner" the example app would then use 3 generators to_planner->iir_blur->to_interleaved?

Reply:
So these are intended to be independent examples, then? Is it useful to have multiple examples for this Rust layer at all? I'd expect the general Halide documentation/examples to be the primary resource a user will draw upon, with a Rust-specific example only necessary to illustrate the Rust portions of the workflow, which I wouldn't expect to change significantly between examples.